Library/Formula/vis.rb

```diff
@@ -0,0 +1,25 @@
+class Vis < Formula
+  desc "Modern, legacy free, simple yet efficient vim-like editor"
+  homepage "http://www.brain-dump.org/projects/vis"
+  url "http://www.brain-dump.org/projects/vis/vis-0.2.tar.gz"
+  sha256 "3e5b81d760849c56ee378421e9ba0f653c641bf78e7594f71d85357be99a752d"
+  head "https://github.com/martanne/vis.git"
+
+  keg_only :provided_by_osx
```

Comment:
Technically it’s not provided by OS X because it’s not the same tool. Something like ``keg_only "Shadows OS X' `vis` command"`` should be more appropriate. See e.g. [`unixodbc.rb`](https://github.com/Homebrew/homebrew/blob/bf20a138788ae2355052619ab59a9b23c3b8e10a/Library/Formula/unixodbc.rb#L21) and [`e2fsprogs.rb`](https://github.com/Homebrew/homebrew/blob/bf20a138788ae2355052619ab59a9b23c3b8e10a/Library/Formula/e2fsprogs.rb#L16).